observatory/landing/form.js

```diff
@@ -0,0 +1,70 @@
+import { LitElement, css, html } from "lit";
+
+export class FormProgress extends LitElement {
+  static styles = css``;
+
+  static properties = {
+    _queryRunning: { state: true, type: Boolean },
+    hostname: { type: String },
```

Comment:
`_hostname: { state: true }` if we only use it internally: or is the idea we might want to prefill the form at some point?